spec/support/shared_examples/api_create_on_resource_endpoint_documentation_support.rb

```diff
@@ -8,13 +8,6 @@
       produces "application/json"
       security [api_key: []]
 
```

Comment:
I think this file should be deleted? it looks like it was replaced/renamed with `create on existing resource`?

Reply:
I split it in two, because one post is for a resource that already exist, and the other one is for a new one. I tried to merge both and the code looked complicated.